index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/estilos-index.css">
+    <link rel="stylesheet" href="/css/flexboxgrid.min.css">
+    <title>CineOnline</title>
+</head>
+<body>
+<div class="container-1">
+        <header>
+            <div>
+                <nav class="row">
+                        <ul class="row col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                            <li class="start-xs start-sm start-md start-lg"><a id="enlace-logo" href="index.html"><img id="logo-cine" src="imagenes/logo-cine.png" alt="Logo CineOnline"></a></li>
+                            <li><h3 id="nombre-cine">CineOnline</h3></li>
+                            <li><a href="index.html">Inicio</a></li>
+                            <li><a href="movies.html">Películas</a></li>
+                            <li><a href="">Series</a></li>
+                            <li><a href="">Precios</a></li>
+                            <li><a href="">Contacto</a></li>
+                            <li class="end-lg"><a href="" id="acceso">Acceder</a></li> 
+                        </ul> 
+                </nav>    
+            </div>
+        </header>
+
+        <div class="row">
+            <div class=" row seccion-uno col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                <div class=" row col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                    <div class="seccion1">
+                        <h1 id="tituloseccion1">Nosotros te ayudamos a encontrar las mejores películas, series y programas de televisión</h1>
+                        <br/>

```

Comment:
Instead of adding space between elements with an element try to add styles. Use `br` element only when its purpose is to break a line.